core/sequencer/sequencer.go

```diff
@@ -277,15 +293,15 @@ func (s *Sequencer) applyMutations(mutations []*tpb.EntryUpdate, leaves []*trill
 }
 
 // CreateEpoch signs the current map head.
-func (s *Sequencer) CreateEpoch(ctx context.Context, forceNewEpoch bool) error {
+func (s *Sequencer) CreateEpoch(ctx context.Context, logID, mapID int64, forceNewEpoch bool) error {
```

Comment:
(pre-existing): I'm not keen on passing bools to funcs that modify behaviour like this.
The callsites where you see CreateEpoch(ctx, 123, 456, true) don't tell you anything and you have to look here to see what true represents.

Reply:
Using a named constant would be better.  Or type an int as Force and define constants for Force.